address_book/api/api.proto

```diff
@@ -5,10 +5,30 @@ option go_package = "address_book/internal/pb";
 import "google/api/annotations.proto";
 
 service AddressBookService {
-  rpc AddContact(AddContactRequest) returns (AddContactResponse) {}
-  rpc FindContact(FindContactRequest) returns (FindContactResponse) {}
-  rpc DeleteContact(DeleteContactRequest) returns (DeleteContactResponse) {}
-  rpc UpdateContact(UpdateContactRequest) returns (UpdateContactResponse){}
+  rpc AddContact(AddContactRequest) returns (AddContactResponse) {
```

Comment:
Since you updated .proto with http paths, please keep in mind that code in your main need to be changed as well to provide http endpoints.

Reply:
Done